src/ucs/sys/string.c

```diff
@@ -127,6 +146,42 @@ void ucs_memunits_to_str(size_t value, char *buf, size_t max)
     }
 }
 
+int ucs_str_to_memunits(const char *buf, void *dest, const void *arg)
+{
+    char units[3];
+    int num_fields;
+    size_t value;
+    size_t bytes;
+
+    /* Special value: infinity */
+    if (!strcasecmp(buf, UCS_CONFIG_NUMERIC_INF_STR)) {
```

Comment:
it's weird that sys/string.c is using constants from config_parser - e.g UCS_CONFIG_NUMERIC_INF_STR
need to move those defs to sys/string.h